system/jlib/jptree.ipp

```diff
@@ -517,15 +517,45 @@ struct AttrStrUnionWithTable : public AttrStrUnion
     }
     static RONameTable *roNameTable;
 };
+
+struct AttrStrUnionWithValueTable : public AttrStrUnionWithTable
+{
+    inline const char *get() const
+    {
+        if (flag==3) // no point in also checking !isPtr() afaics
+            return roValueTable->getIndex(idx2)->str_DO_NOT_USE_DIRECTLY;  // Should probably rename this back now!
+        return AttrStrUnion::get();
```

Comment:
@richardkchapman - btw, do you agree with the comment on line 525?
If so applies to AttrStrUnionWithTable::set() too.